src/main/java/core/basesyntax/dao/jdbc/DaoJdbcManufacturerImpl.java

```diff
@@ -14,23 +14,22 @@
 import java.util.List;
 import java.util.Optional;
 
+@SuppressWarnings("unused")
 @Dao
 public class DaoJdbcManufacturerImpl implements DaoManufacturer {
     @Override
     public Manufacturer create(Manufacturer item) {
         String query = "INSERT INTO manufacturers (manufacturer_name, manufacturer_country) "
                 + "VALUES (?, ?)";
-        try (Connection connection = ConnectionUtil.getConnection()) {
-            PreparedStatement statement
-                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                        = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
             statement.setString(1, item.getName());
             statement.setString(2, item.getCountry());
-            statement.execute();
```

Comment:
does this method work correctly without execute()?

Reply:
No, it's not! Thank you!